src/commonMain/kotlin/com/algolia/search/transport/Host.kt

```diff
@@ -17,6 +17,12 @@ internal val ApplicationID.searchHosts
 
 internal val insightHost = RetryableHost("insights.algolia.io")
 internal val analyticsHost = RetryableHost("analytics.algolia.com")
+internal val placesHost = listOf(
```

Comment:
Would name it plural, as it's not a single host like `insightHost`/`analyticsHost`

```suggestion
internal val placesHosts = listOf(
```

Reply:
Agreed